pkg/services/pluginsintegration/pluginsintegration.go

```diff
@@ -52,7 +52,7 @@ var WireSet = wire.NewSet(
 	coreplugin.ProvideCoreRegistry,
 	pluginscdn.ProvideService,
 	assetpath.ProvideService,
-	angulardetector.ProvideService,
+	angularinspector.ProvideService,
```

Comment:
I would move the service definition to this package. The static detector can stay in the plugins manager as an exposed function. (We may want to change the inspector implementation in other scenarios?)